controls/SV-221586.rb

```diff
@@ -36,4 +36,9 @@
   tag legacy: ["SV-89845","V-75165"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
-end
\ No newline at end of file
+  
+  describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+    it { should have_property '\"AllowDeletingBrowserHistory\"'}
+    its('\"AllowDeletingBrowserHistory\"') { should cmp '\"0\"' }
```

Comment:
The combination of backslash followed by double quotes,`\"`, is not necessary. It checks for `\"AllowDeletingBrowserHistory\"` as opposed to `AllowDeletingBrowserHistory` as the property. Similarly, it checks for `\"0\"` as opposed to `0` as the value. Consider removing this combination `\"` to fix the syntax.